packages/medusa/src/api/routes/store/product-tags/index.ts

```diff
@@ -0,0 +1,45 @@
+import { Router } from "express"
+import { ProductTag } from "../../../../models"
+import { PaginatedResponse } from "../../../../types/common"
+import middlewares, { transformQuery } from "../../../middlewares"
+import { StoreGetProductTagsParams } from "./list-product-tags"
+
+const route = Router()
+
+export default (app: Router) => {
+  app.use("/product-tags", route)
+
+  route.get(
+    "/",
+    transformQuery(StoreGetProductTagsParams, {
+      defaultFields: defaultStoreProductTagFields,
+      defaultRelations: defaultStoreProductTagRelations,
+      allowedFields: allowedStoreProductTagFields,
+      isList: true,
+    }),
+    middlewares.wrap(require("./list-product-tags").default)
+  )
+
+  return app
+}
+
+export const allowedStoreProductTagFields = [
+  "id",
+  "value",
+  "created_at",
+  "updated_at",
+]
+
+export const defaultStoreProductTagFields = [
+  "id",
+  "value",
+  "created_at",
+  "updated_at",
+]
```

Comment:
suggestion: should we always have `defaultFields` as a subset of `allowedFields` and do: 
```
const allowedStoreProductTagFields = [...defaultStoreProductTagFields, ...]
```
to prevent duplication